packages/lambdapi/lambdapi.2.1.0/opam

```diff
@@ -48,13 +48,12 @@ depends: [
   "sedlex" {>= "2.2"}
   "alcotest" {with-test}
   "alt-ergo" {with-test}
-  "alt-ergo" {<= "2.4.0"}
   "bindlib" {>= "5.0.1"}
   "timed" {>= "1.0"}
   "pratter" {>= "1.2"}
   "why3" {>= "1.4.0"}
   "yojson" {>= "1.6.0"}
-  "cmdliner" {>= "1.0.3"}
+  "cmdliner" {>= "1.0.3" & <= "1.0.4"}
```

Comment:
Are you sure? It compiled just fine with cmdliner.1.1.0
see for example: http://check.ocamllabs.io/log/1645246063-968ab5c21b6019c3e65d6b6cbee96fcc8ca43e39/4.14/good/lambdapi.2.1.0

Reply:
Ah, strange, I recently got CI problems with this new version of cmdliner. I am going to check again.

Reply:
It does not work:
```
09:29 ~/src/lambdapi (master) make
File "src/cli/init.ml", line 73, characters 2-11:
73 |   Term.info "init" ~doc
       ^^^^^^^^^
Error (alert deprecated): Cmdliner.Term.info
Use Cmd.info instead.
File "src/cli/install.ml", line 116, characters 2-11:
116 |   Term.info "install" ~doc
        ^^^^^^^^^
Error (alert deprecated): Cmdliner.Term.info
Use Cmd.info instead.
File "src/cli/install.ml", line 121, characters 2-11:
121 |   Term.info "uninstall" ~doc
        ^^^^^^^^^
Error (alert deprecated): Cmdliner.Term.info
Use Cmd.info instead.
make: *** [Makefile:11 : bin] Erreur 1
```

Reply:
these are deprecations. Not real errors. This means you use dune directly, not the build instruction from your opam file

Reply:
I see but it may well be the case that in a few months or years, this version of lambdapi won't compile anymore with cmdliner. So I prefer to anticipate. A PR is already ready for moving to cmdliner 1.1.0: https://github.com/Deducteam/lambdapi/pull/851. So the next release of Lambdapi in a few weeks or months will be for cmdliner >= 1.1.0.